src/modules/rewards/Rewards.tsx

```diff
@@ -88,6 +88,24 @@ const Rewards: FC<RewardsProps> = () => {
   function useQuery() {
     return new URLSearchParams(useLocation().search);
   }
+
+  useEffect(() => {
+    const hash = window.location.hash;
+    if (hash) {
+      const params = new URLSearchParams(hash.substring(1));
+      const token = params.get('access_token');
+      const expiresIn = params.get('expires_in');
+
```

Comment:
@abhishek-01k how we will be having this things in the URL. From where does we get it added in URL?

Reply:
It will be coming for the discord authorisation when the user authorises the dapp it will give a bearer token to the redirect url.